install.sh

```diff
@@ -136,7 +128,6 @@ popd
 
 # Install dependencies
```

Comment:
Please remove the comment and `echo` now that no dependencies are needed anymore